src/Data/Ticket/TicketValuesConfig.ts

```diff
@@ -37,7 +37,46 @@ provideDataValues(Ticket, {
   type: ['PSA_SVC_CAL', 'PSA_SVC_CPL', 'PSA_SVC_TRB'],
 })
 
-provideAttributeLocalizers(Ticket, {
+provideAttributeLocalizers('de', Ticket, {
+  status: {
+    PSA_SVC_C8D_ACQ: 'erfasst',
+    PSA_SVC_C8D_CLS: 'abgeschlossen',
+    PSA_SVC_C8D_DCS: 'entschieden',
+    PSA_SVC_C8D_DON: 'erledigt',
+    PSA_SVC_C8D_DSP: 'disponiert',
+    PSA_SVC_C8D_FLW: 'nachbereitet',
+    PSA_SVC_C8D_NBE: 'nicht begonnen',
+    PSA_SVC_CAL_ACQ: 'erfasst',
+    PSA_SVC_CAL_CLS: 'beantwortet',
+    PSA_SVC_CAL_FRW: 'weitergeleitet',
+    PSA_SVC_CAL_WRK: 'in Bearbeitung',
+    PSA_SVC_CPL_ACQ: 'erfasst',
+    PSA_SVC_CPL_CLS: 'abgeschlossen',
+    PSA_SVC_CPL_DCS: 'Abstellmaßnahme festgelegt',
+    PSA_SVC_CPL_DON: 'Auftrag abgeschlossen',
+    PSA_SVC_CPL_FLW: 'nachbereitet',
+    PSA_SVC_CPL_ORD: 'Auftrag angelegt',
+    PSA_SVC_SUP_CPL_ASQ: 'erfasst',
+    PSA_SVC_SUP_CPL_BEA: 'in Bearbeitung durch Lieferanten',
+    PSA_SVC_SUP_CPL_ERL: 'erledigt',
+    PSA_SVC_SUP_CPL_RUE: 'rückgemeldet durch Lieferanten',
+    PSA_SVC_SUP_CPL_SEN: 'an Lieferanten gemeldet',
+    PSA_SVC_TRB_ACQ: 'erfasst',
+    PSA_SVC_TRB_CLS: 'beseitigt',
+    PSA_SVC_TRB_DCS: 'Abstellmaßnahme festgelegt',
```

Comment:
This status reads strange.

Reply:
According to Ronald these are the localizers from the standard. I don't see any potential todo here.